back/src/Domain/Model/PostTitle.php

```diff
@@ -10,7 +10,9 @@ public function __construct(
         private readonly string $title,
```

Comment:
Naming is better here, even if IMO this kind of value object is a little bit overkill in your case